test/integration/agent-integration.test.ts

```diff
@@ -0,0 +1,44 @@
+import { describe } from 'mocha'
+import * as puppeteer from 'puppeteer'
+import AgentService from '../../src/services/agent-service'
```

Comment:
`AgentService` and `Constants` I think are unused?

Reply:
Oops. Removed.